contrib/postgresql/README.md

```diff
@@ -34,16 +34,24 @@ collectd/postgresql calles the collectd_insert function.
 			double precision[])		-- values
 ```
 
-This seems to represents the user_data_t/notification_t structure.
+This seems to represent the user_data_t/notification_t structure.
 https://github.com/collectd/collectd/blob/ef1e157de1a4f2cff10f6f902002066d0998232c/src/daemon/plugin.h#L172
 
-Lets take the ping plugin as an example. It collects 3 values: ping, ping_stddev, ping_droprate.
+Let's take the ping plugin as an example. It collects 3 values: ping, ping_stddev, ping_droprate.
 
-The current structure creates 3 identifiers and 3 lines for each entry. The identifiers get reused. It reports "192.168.myping.ip" as type.
+The current structure creates 3 identifiers and 3 lines for each entry. The identifiers get reused. It reports "192.168.200.123" as type.
```

Comment:
Where the IP value comes from?

Reply:
From [RFC5737](https://datatracker.ietf.org/doc/rfc5737/):

>   The blocks 192.0.2.0/24 (TEST-NET-1), 198.51.100.0/24 (TEST-NET-2), and 203.0.113.0/24 (TEST-NET-3) are provided for use in documentation.

Reply:
Suggestion: spell out what is being reported here and provide an example. The quotes make it look like this is the literal IP address that is being reported each time.

“The ping plugin reports an IP address in the type instance field, e.g. "192.0.2.42".”